examples/mockups/index.html

```diff
@@ -188,11 +188,11 @@ <h2 class="ui header">How it works</h2>
           
               <div class="ui basic vertical huge segment">
                 <p>
-                  We generate, host and manage your documentation versions for you, outputting both html and pdf formats. 
+                  We generate, host and manage your documentation versions for you, shipping both HTML and PDF. 
```

Comment:
"Outputting" sounded weird, hope this is better?

Reply:
I like shipping more than outputting. We could also move this around to drop the verb: "We can generate HTML, PDF, and ePUB formats for your project, and will host and manage ..."

Reply:
I don't like shipping, it feels pretty jargony. I'd use `creating` or `generating` as suggested.

Reply:
This line starts with `generate` already, so `generating` is redundant, but otherwise I prefer this verb. My first vote is still restructuring the sentence to drop the double verbs altogether and make the sentence clearer.

```suggestion
                  We generate HTML, PDF, and ePUB formats for your project, and host and manage documentation versions for you.
```